### PR TITLE
fix(member): cannot remove member from voice channel

### PR DESCRIPTION
**Pull request by GeomKid**
## About

This pull request is about (X), which does (Y).

## Checklist

- [x] The ``pre-commit`` code linter has been run over all edited files to ensure the code is linted.
- [x] I've ensured the change(s) work on `3.8.6` and higher.
- [ ] I have added the `versionadded`, `versionchanged` and `deprecated` to any new or changed user-facing function I committed.

### Pull-Request specification

I've made this pull request: (check all that apply)
  - [x] For the documentation
  - [ ] To add a new feature
  - [ ] As a general enhancement
  - [ ] As a refactor of the library/the library's code
  - [ ] To fix an existing bug
  - [ ] To resolve #ISSUENUMBER

This is:
  - [ ] A breaking change

### CODE REVIEW

File: interactions/api/models/member.py [hunk 1]
```diff
@@ -480,7 +480,7 @@ async def modify(
         roles: Optional[List[int]] = MISSING,
         mute: Optional[bool] = MISSING,
         deaf: Optional[bool] = MISSING,
-        channel_id: Optional[Union[Channel, int, Snowflake]] = MISSING,
+        channel_id: Optional[Union[Channel, int, Snowflake, None]] = MISSING,
```
Comment (Damego) [inline interactions/api/models/member.py]: It's already typehinted as optional

File: interactions/client/context.py [hunk 2]
```diff
@@ -199,16 +199,15 @@ async def defer(
                 application_id=int(self.id),
                 data={"type": self.callback.value, "data": {"flags": is_ephemeral}},
             )
-
             with suppress(LibraryException):
                 res = await self._client.get_original_interaction_response(
                     self.token, str(self.application_id)
                 )
-                self.message = Message(**res, _client=self._client)
+                _message = Message(**res, _client=self._client)
```
Comment (Damego) [inline interactions/client/context.py]: It's should not be here

Comment (GeomKid) [reply interactions/client/context.py]: Hmm somehow my branching got problem I'll fix that

**Comment by Damego** [comment]
You forgot to remove `None` in the params

**Review by Damego — APPROVED** [review]

**Pull request merged**